### ISSUE TITLE
The 'pulls' counter is incremented by 4 per each pushed tag

### ISSUE THREAD

**Issue by Javernaut** [docker/hub-feedback#2182]
While trying to automate an image creation and pushing on Github Actions I faced a weird behaviour of the 'pulls' counter of the Docker Hub repository page. I assume it is incremented by 1 per each push action. But it was actually incremented by 4 per each tag that I pushed. I tried it both with public and private pairs of Github and Docker Hub repositories. Also, I tried pushing from a local machine to both public and private Docker Hub repositories, and the result was the same.

Here are the steps to preproduce:
1. Prepare a Docker Hub repo
2. Create an image for it and tag it
3. Visit the public page of the repo and notice the 'pulls' counter
4. Push the image
5. Refresh the public page of the repo. Notice the counter is increased by just 1
6. **Refresh the page again. Notice the counter is increased additionally by 3**

Further refreshing doesn't affect the counter. It doesn't matter how much time passed since step 4. If the pushed image had multiple tags, then the numbers in steps 5 and 6 would be multiplied by the number of tags pushed.

It is super weird and I would like this issue to be investigated.

Here are the public pair of [Github](https://github.com/Javernaut/DockerPullsCounterTest) and [Docker Hub](https://hub.docker.com/r/javernaut/docker-pulls-counter-test/tags) repositories. The Docker version used in Github Actions environment is: `Docker version 20.10.11+azure-3, build dea9396e184290f638ea873c76db7c80efd5a1d2`. The CI job is triggered when a tag pushed to Github repo has `v*.*.*` pattern.

On my local machine with macos I use this version of Docker: `Docker version 20.10.11, build dea9396`.

**Comment by ingshtrom** [comment]
Do you happen to have vulnerability scanning enabled on these repos?

**Comment by Javernaut** [comment]
@ingshtrom No, I didn't enable it. I literally did no configuration of the repositories.

**Comment by Javernaut** [comment]
Additional note. 
I also tried executing the docker commands on CI directly for building, logging in and pushing instead of using the actions. So the implementation of these actions shouldn't be a concern. 
Also the fact that doing the very same on my local machine still reproduces the issue confirms that fact.

**Comment by ingshtrom** [comment]
👍  just checking. Thank you the response and the report. We have reproduced this and are looking for the root cause.

**Comment by mortymacs** [comment]
Hi,
Recently, we've found is that the number of Docker pull is not increasing anymore. Is it related to this thread?
The Docker image that I'm tracking is `atkrad/wait4x`.
